lang/en.json

```diff
@@ -106,7 +106,7 @@
         "ComputedMetricCountWithValueDocumentation": "The number of entries that have a value set for %s",
         "ConfigFileIsNotWritable": "The Piwik configuration file %1$s is not writable, some of your changes might not be saved. %2$s Please change permissions of the config file to make it writable.",
         "Continue": "Continue",
-        "ContinueToPiwik": "Continue to Piwik",
+        "ContinueToPiwik": "Upgrade and continue to Piwik",
```

Comment:
This translation text is also used in Installation. Imho it wouldn't make sense to show "upgrade and continue" there. Maybe we would need to introduce a new translation text.